File: Python/Math-Array-String/77.py
# 1832. Check if the Sentence Is Pangram

class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        return len(set(sentence)) == 26
    
sol = Solution()
output = sol.checkIfPangram("abccbaacz")
print(output)  # Output: False

File: JavaScript/Math-Array-String/77.js
// 1832. Check if the Sentence Is Pangram

/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    return new Set(sentence).size === 26;
};

// Example usage:
console.log(checkIfPangram("thequickbrownfoxjumpsoverthelazydog")); // Output: true
console.log(checkIfPangram("leetcode")); // Output: false

